### PR TITLE
JDK-8315409: Fix jfx-tests so they work with latest jfx build

### PR DESCRIPTION
**Pull request by shurymury**
JDK-8315409: Fix jfx-tests so they work with latest jfx build

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed (2 reviews required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer), 1 [Author](https://openjdk.org/bylaws#author))

### Issue
 * [JDK-8315409](https://bugs.openjdk.org/browse/JDK-8315409): Fix jfx-tests so they work with latest jfx build (**Bug** - P4)

### Reviewers
 * [Kevin Rushforth](https://openjdk.org/census#kcr) (@kevinrushforth - **Reviewer**)
 * [Ajit Ghaisas](https://openjdk.org/census#aghaisas) (@aghaisas - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jfx-tests.git pull/1/head:pull/1` \
`$ git checkout pull/1`

Update a local copy of the PR: \
`$ git checkout pull/1` \
`$ git pull https://git.openjdk.org/jfx-tests.git pull/1/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1`

View PR using the GUI difftool: \
`$ git pr show -t 1`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jfx-tests/pull/1.diff">https://git.openjdk.org/jfx-tests/pull/1.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jfx-tests/pull/1#issuecomment-1701738789)

### CODE REVIEW

**Comment by kevinrushforth** [comment]
Normally, a single reviewer would be sufficient (as with most test bugs), but for this initial fix, I'd like @aghaisas to sanity check it as well.

/reviewers 2

File: bigapps/SceneBuilderTest/src/scenebuilder/RunGUIBrowser.java [hunk 1]
```diff
@@ -1,10 +1,18 @@
 package scenebuilder;
```
Comment (kevinrushforth) [inline bigapps/SceneBuilderTest/src/scenebuilder/RunGUIBrowser.java:25]: Missing copyright header.

File: functional/ControlsTests/src/javafx/factory/ControlsFactory.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2014, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2014, 2023 Oracle and/or its affiliates. All rights reserved.
```
Comment (kevinrushforth) [inline functional/ControlsTests/src/javafx/factory/ControlsFactory.java]: You need a comma after `2023`.

File: tools/Jemmy/GlassImage/src/org/jemmy/image/glass/ClasspathGlassImageLoader.java [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2009, 2012, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
```
Comment (kevinrushforth) [inline tools/Jemmy/GlassImage/src/org/jemmy/image/glass/ClasspathGlassImageLoader.java]: Shouldn't the original copyright year of 2009 be retained?

**Review by kevinrushforth — COMMENTED** [review]
The changes look reasonable to me. I'll rely on Ajit to sanity test it.

I did note a few copyright issues that you might want to look at. At a minimum, the missing comma after the `2023` is needed everywhere.

There are a few files without copyright headers, but many are preexisting, so unless you are otherwise touching those files, they can be done later.

**Comment by aghaisas** [comment]
I verified that the following tests can be run now with the latest jfx build.

Tests that run with the latest jfx build (some test failures are observed fixing which is out of scope of this PR) :
- 3DTests
- ControlsTests
- FxmlTests
- SceneGraphTests


Tests that do not compile yet (I believe, this is future work) :
- WebNodeManualTests
- ControlsLeakTests
- FXCssTests
- WebNodeAutomated
- appBundlerTestsJDK9

**Review by aghaisas — APPROVED** [review]
Approving this PR as it will serve as the foundation of future fixes.

**Review by kevinrushforth — APPROVED** [review]
There are still 122 more files that are missing a `,` after `2023`. I will file a follow-on issue so as not to delay getting this in. We can run our copyright year update script to fix the rest.

**Comment by kevinrushforth** [comment]
I just now ran our copyright script and submitted PR #2 to fix the copyright lines.

**Pull request closed without merging**